### DISCUSSION TITLE
moving Bayesian dependencies to sktime-Bayes

### DISCUSSION THREAD

**Discussion by fkiraly** [alan-turing-institute/sktime#952] [Ideas]
`prophet` is making set-up sometimes difficult - building wheels on windows is no joke.

Generally, should we think about moving the "Bayesian" dependenies (such as `pystan`, `pymc3` etc) out into a companion package `sktime-Bayes`? Just like we have neural networks in `sktime-dl`.

**Comment by fkiraly** [comment]
@mloning, @aiwalter?

**Comment by mloning** [comment]
Yes, that would be my preferred solution. To do this well we need:

* a good template repository for sktime companion packages,
* expose our API relevant unit tests so that they can be run easily from within other repositories.
